src/recode.jl

```diff
@@ -11,6 +11,8 @@ or [`in`](@ref) the key (first item of the pair), then the corresponding value
 (second item) is copied to `dest`.
 If the element matches no key and `default` is not provided or `nothing`, it is copied as-is;
 if `default` is specified, it is used in place of the original element.
+If `dest` is CategoricalArray and `default` is provided then the ordering of resulting levels
```

Comment:
Good idea. But the order of pairs is always used for the levels IIRC, it's just that when `default` is not provided original levels appear first. I guess you can just omit "and `default` is provided" but say "`default` will be the last level if provided".

Also use backticks around `CategoricalArray`, and I'd say "a `CategoricalArray`. Finally, I think it would make sense to mention this only for a `CategoricalArray` method, after showing its signature, just like it's currently done for arrays supporting missing values.

Reply:
fixed